strings-csharp/StringTruncationInCSharp/StringTruncationInCSharp/StringHelperBenchmark.cs

```diff
@@ -0,0 +1,84 @@
+        }
+
+
+        [Benchmark]
+        public void TruncateStringUsingSubstring()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
+
+            _stringHelper.TruncateStringUsingSubstring(_originalString, maxLength);
+        }
+
+        [Benchmark]
+        public void TruncateStringUsingForLoop()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
+
+            _stringHelper.TruncateStringUsingForLoop(_originalString, maxLength);
+        }
+
+        [Benchmark]
+        public void TruncateStringUsingForLoopWithStringBuilder()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
+            _stringHelper.TruncateStringUsingForLoopWithStringBuilder(_originalString, maxLength);
+        }
+
+        [Benchmark]
+        public void TruncateStringUsingRegularExpressions()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
+            _stringHelper.TruncateStringUsingRegularExpressions(_originalString, maxLength);
+        }
+
+        [Benchmark]
+        public void TruncateStringUsingRemove()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
+
+            _stringHelper.TruncateStringUsingRemove(_originalString, maxLength);
+        }
+
+        [Benchmark]
+        public void TruncateStringUsingLINQ()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
+
+            _stringHelper.TruncateStringUsingLINQ(_originalString, maxLength);
+        }
+
+        [Benchmark]
+        public void TruncateStringUsingSpan()
+        {
+            int maxLength = (int)(_originalString.Length * MaxLengthPercentage);
+
+            _stringHelper.TruncateStringUsingSpan(_originalString, maxLength);
+        }
+
+        private string GetDummyStringToTestBenchmark()
+        {
+            var dummyText = "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate eleifend tellus. Aenean leo ligula, porttitor eu, consequat vitae, eleifend ac, enim. Aliquam lorem ante, dapibus in, viverra quis, feugiat a, tellus. Phasellus viverra nulla ut metus varius laoreet. Quisque rutrum. Aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi. Nam eget dui. Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum. N";
```

Comment:
Since we're not interested in chars in the string at all (we only care about it's length) it's better to use new string('a', 1000). And this should be assigned directly in the `_originalString` variable